### DISCUSSION TITLE
[Mining Issue] miner-init takes over a day and not progress updates or completion

### DISCUSSION THREAD

**Discussion by jimzen** [filecoin-project/lotus#5947] [General Q&A] [answered]
lotus-miner stuck at following without any updates:

**Describe the problem**
A brief description of the problem you encountered while mining new blocks.

State: complete; Current Epoch: 633001; Todo: 0

Done!
2021-04-01T17:00:32.492Z        INFO    main    lotus-storage-miner/init.go:166 Checking if repo exists
2021-04-01T17:00:32.492Z        INFO    main    lotus-storage-miner/init.go:182 Checking full node version
2021-04-01T17:00:32.493Z        INFO    main    lotus-storage-miner/init.go:193 Initializing repo
2021-04-01T17:00:32.493Z        INFO    repo    repo/fsrepo.go:122      Initializing repo at '/home/zenlayer/fil_para/miner'
2021-04-01T17:00:32.494Z        INFO    main    lotus-storage-miner/init.go:408 Initializing libp2p identity
2021-04-01T17:00:32.506Z        INFO    badger  v2@v2.2007.2/levels.go:183      All 0 tables opened in 0s

2021-04-01T17:00:32.522Z        INFO    badger  v2@v2.2007.2/levels.go:183      All 0 tables opened in 0s

2021-04-01T17:00:37.866Z        INFO    main    lotus-storage-miner/init.go:700 Pushed CreateMiner message: xxxxx
2021-04-01T17:00:37.867Z        INFO    main    lotus-storage-miner/init.go:701 Waiting for confirmation

And now it is one day after and still like that when height is 635920

**Version**

The output of `lotus --version`.
ze:~$ lotus --version
lotus version 1.6.0-dev+mainnet+git.6d9ae1e28

**Setup**

This is a ice lake server, 256G memory and 3T storage. Daemon sync is completed:

	Base:	[bafy2bzacecfulr35ntw46qrpuecdkn2dtg6zbffmkv4ne4iolouw57jxz5mw2 bafy2bzaceagd5piv452ap5ksj7qtybku3rz2lli3l3kwvnis3s54cs2mn6hke bafy2bzaceap4vujeoixfifoijsr6h4qp5qgxzk2stgghs6fapp6qktm2il2aa bafy2bzacebxnkdf2qllstg44i6bvona37bdmmaocwnilex3qzldcl4bzkst6o bafy2bzaceclfsz2sjqmmsv4sasvtpmxov7ltima32s7i6jfn4rqr4nlrtgnym]
	Target:	[bafy2bzaceawgtkto5ewci6avpnrsaxge6smaizeb6wwstcmqnqafxjyi4gt4k] (635920)
	Height diff:	0
	Stage: complete
	Height: 635920

**Comment by jennijuju** [comment, marked as answer]
This likely is caused due to the message is stuck in mpool. Try following the steps [here](https://docs.filecoin.io/mine/lotus/message-pool/#replacing-messages-in-the-pool) to get it push through

**Reply by jimzen** [reply]
I adjusted the gas-premium and gas-feecap to a high value comparing with base, but still no luck:
lotus mpool replace --gas-feecap 5555236571 --gas-premium 1299953 f3xxxxx  6
or lotus mpool replace --auto 1299953 f3xxxxx  6
while the FIL send can go through without specifying fees

basefee:
lotus chain head | xargs lotus chain getblock | jq -r .ParentBaseFee
1025642326

Do I need to adjust the gaslimit as well?

**Reply by jimzen** [reply]
lotus mpool pending --local
{
  "Message": {
    "Version": 0,
    "To": "f04",
    "From": "f3xxxxxx",
    "Nonce": 7,
    "Value": "0",
    "GasLimit": 60000000,
    "GasFeeCap": "15555236571",
    "GasPremium": "451000",
    "Method": 2,
    "Params": "hxxxxx==",
    "CID": {
      "/": "bafxxxxxx"
    }
  },

**Reply by jimzen** [reply]
Finally, after 4 more hours. The init is passed.

**Comment by 2Spartiate** [comment]
Hi,

@jennijuju 

I have the same problem, and this solution doesn't seem to work with me:
- I kept the first ssh window running on "Waiting for confirmation", and oppened a new one
- on the new one I entered: 
sudo  lotus mpool replace --auto f3xxxxx 1
- got: "new message cid:  bafy2"

And nothing happenned on window 1 (stucked since about 40 hours on the same message..)
I have waited 4 more hours, and nothing more. So I have restarted the init command, and even tried nonce at 0 (the nounce number of the owner's address), nothing.

What am i doing wrong ?


$ sudo lotus mpool pending --local

  "Message": {
    "Version": 0,
    "To": "f04",
    "From": "f3xxxx",
    "Nonce": 0,
    "Value": "0",
    "GasLimit": 44824342,
    "GasFeeCap": "156165147",
    "GasPremium": "124898",
    "Method": 2,
    "Params": "hVgxxxxx",
    "CID": {
      "/": "bafy2xx"
    }
  },
